dnf/subject.py

```diff
@@ -148,3 +148,19 @@ def get_best_selector(self, sack, forms=None):
             dep = str(reldep)
             return sltr.set(provides=dep)
         return sltr
+
+    def get_best_selectors(self, sack, forms=None):
+        if not self.filename_pattern and is_glob_pattern(self.subj.pattern):
+            nevra = first(self.subj.nevra_possibilities_real(sack, allow_globs=True))
+
+            if nevra and nevra.name and is_glob_pattern(nevra.name):
+                    sltrs = []
+                    pkgs = sack.query().filter(name__glob=nevra.name)
```

Comment:
Just have seen some scenario in other bug report. There should be `nevra.to_query()` instead. If you had available `pkg A.x86_64, pkg A-lib.i686` then `dnf install A*.x86_64` would fail with error like: `can't install A-lib.x86_64`